manifests/k/KsplDev/Net8TestApp/1.0.0.0/KsplDev.Net8TestApp.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KsplDev.Net8TestApp
+PackageVersion: 1.0.0.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Net8TestApp.exe
+Installers:
+- InstallerUrl: https://github.com/KsplDev/Net8TestApp/releases/download/TestRelease/Net8TestApp.zip

```

Comment:
I don't think a `TestRelease` makes sense to put into WinGet. Is there a stable release that isn't a test release?